test/first-order/no-arguments.coffee

```diff
@@ -64,16 +64,29 @@ module.exports = async ->
 
   run "sass"
 
+  define "handlebars", async ->
+    yield go [
+      glob "*.{hb,hbs,handlebars}", src
```

Comment:
I agree with this. I was trying to suggest that we make it so that h9 would work with all the different file extensions. English is hard sometimes...